boilerplates/react-union-boilerplate-liferay-basic/liferay/content-tag-portlet/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,11 @@
+<%--
+Renders React widgets.
+--%>
+<%@ include file="./init.jsp" %>
+
+<%--suppress JSUnresolvedVariable, JSUnresolvedFunction --%>
+<script type="text/javascript">
```

Comment:
I would like to see the script in the tag react-union:widget. Can't we place script here, to make final code cleaner?

Reply:
I added react-union:app tag that is responsible for loading JS application module.